### PR TITLE
fix forms

### PR DESCRIPTION
**Pull request by piensaenpixel**
hey @nobuti could you take a look? Could be great if you can test in local sorry but I ❤️  you

This PR fix  https://github.com/CartoDB/cartodb/issues/9591 and this one https://github.com/CartoDB/cartodb/issues/9594

### CODE REVIEW

File: app/assets/stylesheets/editor-3/forms.css.scss [hunk 1]
```diff
@@ -7,17 +7,16 @@
 
 .Editor-formInner {
   @include display-flex();
-  @include align-items(baseline, baseline);
+  //@include align-items(baseline, baseline);
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/forms.css.scss]: I guess we can remove the comment

File: app/assets/stylesheets/editor-3/forms.css.scss [hunk 2]
```diff
@@ -26,6 +25,7 @@
 }
 .Editor-formInner--noMargin {
   margin-top: 0;
+  padding-top: 8px;
```
Comment (nobuti) [inline app/assets/stylesheets/editor-3/forms.css.scss]: var?

**Comment by nobuti** [comment]
Code is 💯, waiting for staging to review it visually.

**Comment by nobuti** [comment]
🇪🇸

**Comment by matallo** [comment]
bump `package.json`?

**Pull request merged**